codecov_cli/commands/labelanalysis.py

```diff
@@ -285,20 +285,20 @@ def _send_labelanalysis_request(payload, url, token_header):
     return eid
 
 
-def _dry_run_output(result: LabelAnalysisRequestResult):
-    logger.info(
-        "Not executing tests because '--dry-run' is on. List of labels selected for running below."
-    )
-    logger.info("")
-    logger.info("Label groups:")
-    logger.info(
-        "- absent_labels: Set of new labels found in HEAD that are not present in BASE"
+def _dry_run_output(
+    result: LabelAnalysisRequestResult, runner: LabelAnalysisRunnerInterface
+):
+    labels_to_run = list(
+        set(
+            result.absent_labels
+            + result.global_level_labels
+            + result.present_diff_labels
+        )
     )
-    logger.info("- present_diff_labels: Set of labels affected by the git diff")
-    logger.info("- global_level_labels: Set of labels that possibly touch global code")
-    logger.info("- present_report_labels: Set of labels previously uploaded")
-    logger.info("")
-    logger.info(json.dumps(result))
+    output = runner.dry_run_runner_options + sorted(labels_to_run)
+    with open("ATS_TESTS_TO_RUN", "w") as fd:
+        fd.write(" ".join(output) + "\n")
+    click.echo(f"ATS_TESTS_TO_RUN=\"{' '.join(output)}\"")
```

Comment:
it would be really nice from a consumer perspective to receive just a comma-separated list

Reply:
I don't see how a comma-separated list is better than a space-separated list. Especially if you consider that command line args are a space separated list, doing it comma separated seems more work.

If you are referring to `ATS_TESTS_TO_RUN` I added that to make it easier to `grep` that line in the output. There will be other output lines, so it makes it easier to find the right one.

Reply:
Maybe we just output to stdout by default but can send to a file with `--ouput-path={path}` or something?  The hardcoded filename feels a little weird to me here.

Reply:
hum... that's a valid point. If they specify `--output-path={path}` I guess we can also infer that it's `--dry-run` so users don't have to pass both options, right?

Reply:
I guess I was thinking you'd specify both just in case there's another command that uses `--output-path` for something else someday.

Reply:
Well options are per command so even if another command uses that some day it won't matter much. 
If it needs to be tied to `dry-run` I think it should be `dry-run-output-path`
Thoughts on naming?

Reply:
I like the explicitness of 2 flags personally but I think either way is fine.